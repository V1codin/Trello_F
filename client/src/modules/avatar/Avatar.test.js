import React from "react";
import { render, screen } from "@testing-library/react";
import { Avatar } from ".";

describe("Avatar", () => {
  it("renders without crashing", () => {
    const { asFragment } = render(<Avatar />);

    expect(asFragment(<Avatar imgLink="test img link" />)).toMatchSnapshot();
  });
});
