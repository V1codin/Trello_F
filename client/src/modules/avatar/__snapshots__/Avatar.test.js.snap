// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="avatar sb-avatar"
    style="display: inline-block; vertical-align: middle; width: 40px; height: 40px; border-radius: 100%; font-family: Helvetica, Arial, sans-serif;"
  >
    <div
      class="avatar sb-avatar__text"
      style="width: 40px; height: 40px; line-height: initial; text-align: center; color: rgb(255, 255, 255); border-radius: 100%;"
    >
      <div
        style="display: table; table-layout: fixed; width: 100%; height: 100%;"
      >
        <span
          style="display: table-cell; vertical-align: middle; font-size: 100%; white-space: nowrap;"
        >
          <span />
        </span>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
